packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.styles.ts

```diff
@@ -150,6 +150,21 @@ export const getMenuItemStyles = memoizeFunction((
       verticalAlign: 'middle',
       flexShrink: '0',
       fontSize: FontSizes.mini
+    },
+    splitDivider: {
```

Comment:
I noticed this was a pattern in my code a lot to get a divider to line up properly. I created this PR to help reduce duplication. Once that gets merged can you update the PR to use that component instead (can be after this PR goes in since the other one might take longer). https://github.com/OfficeDev/office-ui-fabric-react/pull/3115